tox.ini

```
@@ -1,49 +1,55 @@
 [tox]
 envlist =
-    {py35}-{2.1,2.2}
-    {py36}-{2.1,2.2}
-    {py37}-{2.1,2.2}
-    flake8
+    {py35}-{django21,django22}
+    {py36}-{django21,django22,django30,djangomaster}
+    {py37}-{django21,django22,django30,djangomaster}
+    {py38}-{django30,djangomaster}
+    lint
     docs
-    black
 
 [testenv]
 setenv =
-    PYTHONWARNINGS=module::DeprecationWarning
+    PYTHONWARNINGS=all
+whitelist_externals = make
 commands =
-    coverage run --source=bootstrap4 runtests.py
-    coverage report -m
+    coverage run --source bootstrap4 manage.py test -v1 --noinput
 deps =
     coverage
-    beautifulsoup4
-    2.1: django>=2.1,<2.2
-    2.2: django>=2.2,<2.3
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
+    django30: Django>=3.0,<3.1
```

This (slightly cleaner) syntax is also supported: 

```suggestion
    django21: Django==2.1.*
    django22: Django==2.2.*
    django30: Django==3.0.*
```

https://github.com/jieter/django-tables2/blob/master/tox.ini#L27-L29

---

Nice, I'll use this.